Automattic/pocket-casts-android
# Add a setting for adjusting remaining time

## Description

We got a feedback [here](https://github.com/Automattic/pocket-casts-android/pull/2955#issuecomment-2421956935) that this should be a setting.

I'm targeting beta since I suspect that a lot of users might be confused about this change when enabled by default.

## Testing Instructions

1. Play something with chapters.
2. Change playback speed.
3. Deselect some chapters.
4. Check that the player doesn't display remaining time as adjusted time.
5. Go to `Profile -> Settings -> General`.
6. Enable `Adjust remaining time` setting.
7. Check that the player displays adjusted time as the remaining time.
8. Disable `Adjust remaining time` setting.
9. Check that the player no longer displays adjusted time as the remaining time.

## Checklist
